### PR TITLE
Admin docs update

### PR DESCRIPTION
**Pull request by pameladelgado**
Changed some wordings and added some more links for official documentation (prerequisites).
Depends on https://github.com/SwissDataScienceCenter/renku-admin-docs/pull/12 : get the right reference to some files.

### CODE REVIEW

File: docs/admin/prerequisites/nginx.rst [hunk 1]
```diff
@@ -1,8 +1,10 @@
 .. _nginx:
 
-NGINX ingress configuration
+NGINX ingress
 ============================
 
+Renku provides some `kubernetes services <https://kubernetes.io/docs/concepts/services-networking/service/>`_ that need to be exposed. Kubernetes uses NGINX Ingress for managing routes and controlling traffic.
```
Comment (rokroskar) [inline docs/admin/prerequisites/nginx.rst]: it's not the kubernetes uses nginx for ingress by default, but we happen to provide a setup which assumes an nginx ingress. Maybe this should be reworded slightly to reflect this fact.

File: docs/admin/prerequisites/tiller.rst [hunk 2]
```diff
@@ -3,10 +3,25 @@
 Helm
 ===================
```
Comment (rokroskar) [inline docs/admin/prerequisites/tiller.rst]: probably need to shorten this underline

File: docs/admin/prerequisites/tiller.rst [hunk 3]
```diff
@@ -3,10 +3,25 @@
...
+Renku is deployed via Helm.
 
-To be able to run helm commands (how Renku is deployed) you need to have tiller running in your k8s cluster.
-For deploying tiller you can run the following commands, using the renku-admin-docs `tiller-rbac-config.yaml file <https://github.com/SwissDataScienceCenter/renku-admin-docs/blob/master/helm-installs/tiller-rbac-config.yaml>`_
+`Helm <https://helm.sh/>`_ is a tool used to install and manage Kubernetes applications.
+
+Helm has two parts: a client side (helm) and a server side (tiller).
+
+Helm client
+------------------------------------
```
Comment (rokroskar) [inline docs/admin/prerequisites/tiller.rst]: shorten the underline

File: docs/admin/prerequisites/tiller.rst [hunk 4]
```diff
@@ -3,10 +3,25 @@
...
+
+To install helm client in your laptop, a cluster VM or wherever you want to run the helm commands from, please refer to `Helm documentation <https://helm.sh/docs/using_helm/#installing-the-helm-client>`_ and `Helm github repository <https://github.com/helm/helm#install>`_.
+
+Tiller
+------------------------------------
```
Comment (rokroskar) [inline docs/admin/prerequisites/tiller.rst]: shorten

File: docs/admin/prerequisites/nginx.rst (hunk 1 above)
Comment (pameladelgado) [reply docs/admin/prerequisites/nginx.rst]: :+1:  will rephrase, thanks!

**Pull request merged**